public/components/Notifications/NotificationForm.tsx

```diff
@@ -61,6 +66,13 @@ export const NotificationForm: React.FC<NotificationFormProps> = ({
       if (matchingChannel) selectedNotificationChannelOption.push(matchingChannel);
     });
   }
+  let preview = '';
+  try {
+    preview = `${Mustache.render(action?.subject_template.source, context)}\n\n${Mustache.render(action?.message_template.source, context)}`;
+  } catch (err) {
+    preview = err.message;
+    console.error('There was an error rendering mustache template', err);
```

Comment:
Do you think it will be useful to show this error to the user using a toast notification instead?

Reply:
We are already showing error as message preview, which should be informative to users. I don't see toasts being actively used in this package. If needed at multiple places, we can add the dependency later.
Enhanced error message though.